database/seeders/DatabaseSeeder.php

```diff
@@ -111,6 +111,9 @@ public function run(): void
             ...AssetSeeder::metadataSeeders(),
             AssetSeeder::class,
             MaintenanceProviderSeeder::class,
+            AnalyticsResourceSourceSeeder::class,
+            AnalyticsResourceCategorySeeder::class,
```

Comment:
Let's add `AnalyticsResourceSourceSeeder` and `AnalyticsResourceCategorySeeder` to the Demo database seeder as well.